src/Storage/Notification.php

```diff
@@ -0,0 +1,200 @@
+<?php
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Storage;
+
+use Google\Cloud\Core\ArrayTrait;
+use Google\Cloud\Core\Exception\NotFoundException;
+use Google\Cloud\Storage\Connection\ConnectionInterface;
+
+/**
+ * Cloud Pub/Sub Notifications sends information about changes to objects in
```

Comment:
```diff
- sends
+ send
```

Alternative suggestion for the first sentence of this paragraph:

> Cloud Pub/Sub notifications publish changes to objects in a Cloud Storage bucket as messages in a Cloud Pub/Sub topic of your choice.

Reply:
This is directly from https://cloud.google.com/storage/docs/pubsub-notifications